lib/core/src/sdk.rs

```diff
@@ -181,25 +184,14 @@ impl LiquidSdkBuilder {
             &fingerprint_hex,
         )?;
 
-        let sync_enabled = self
-            .config
-            .sync_service_url
-            .clone()
-            .map(|_| true)
-            .unwrap_or(false);
-
         let persister = match self.persister.clone() {
             Some(persister) => persister,
-            None => {
-                let persister = Arc::new(Persister::new(
-                    &working_dir,
-                    self.config.network,
-                    sync_enabled,
-                )?);
-                persister.init()?;
-                persister.replace_asset_metadata(self.config.asset_metadata.clone())?;
-                persister
-            }
+            None => Arc::new(Persister::new_using_fs(
```

Comment:
Won't using `Persister::new_using_fs()` cause compilation issues in WASM?

Reply:
Yes, it isn't even included in wasm builds. The idea is that wasm builds will always use `new_in_memory()`.

Reply:
How is this going to compile then if wasm calls the `build()` fn?

Reply:
You're right. I'll fix it later today or tomorrow. Thanks for spotting that. There might be some more similar issues with WASM builds on these PRs I've prepared. WASM can only be built when the esplora PR and some of them are put together, so I haven't built them and was aware I could be missing things like this.

As long as native builds are good, we should be fine for now with potential WASM issues, which we'll fix once we have all PRs merged.

Reply:
I added conditional compilation imposing the use of a custom persister on WASM builds to address the issue.